### DISCUSSION TITLE
Props type definition of key "items" and "multiple" Omitted on multiple components

### DISCUSSION THREAD

**Discussion by tokidoki11** [vuetifyjs/vuetify#16911] [Q&A] [answered]
Hi, Im currently building a storybook with vuetify 3

when creating V-Select component, defining argtypes (refer to V-Select props)
I couldnt use multiple and items typings

upon further check, I notice that multiple and items keys are being omitted on not only V-Select but other components as well

<img width="736" alt="image" src="https://user-images.githubusercontent.com/7564844/225198590-2d02dc36-317c-453e-8f31-f62488d6e0c8.png">

Is there any idea behind this?

**Comment by KaelWD** [comment, marked as answer]
IDK what argtypes is. Those props are omitted because they're defined in the generic further down instead.

**Reply by tokidoki11** [reply]
@KaelWD understandable

further down to what component?

FYI
argtypes is to define props in storybook https://storybook.js.org/docs/7.0/vue/writing-stories/args

currently looking at this, it has this definition, because of the omit
importing them will alert the ts server
<img width="732" alt="image" src="https://user-images.githubusercontent.com/7564844/225216394-a663651c-e911-4915-a1e7-0ed71dceded7.png">
